src/ResourceManagement/HDInsight/HDInsight.Tests/ScenarioTests/CreateClusterTests.cs

```diff
@@ -235,6 +235,7 @@ public void TestCreateLinuxClusterWithPremiumTier()
                 var resourceGroup = HDInsightManagementTestUtilities.CreateResourceGroup(resourceManagementClient);
 
                 var cluster = GetClusterSpecHelpers.GetCustomCreateParametersIaas();
+                cluster.Version = "3.4";
```

Comment:
This test failed when I was rerecording the tests because it was using the wrong version of HDInsight that did not support the Premier tier